lib/core/topologies/read_preference.js

```diff
@@ -77,8 +81,8 @@ const VALID_MODES = [
   ReadPreference.SECONDARY,
   ReadPreference.SECONDARY_PREFERRED,
   ReadPreference.NEAREST,
-  true,
-  false,
+  // true,
```

Comment:
should these be deleted

Reply:
also, does removing these break any external workflows?

Reply:
I don't think so and I think that `true` and `false` should not be considered valid `ReadPreference` modes anyways.